### PR TITLE
docs: replace `docs.ipfs.io` with `docs.ipfs.tech`

### PR DESCRIPTION
**Pull request by dbadoy**
The PR to fix the redirecting URL in README.md

### CODE REVIEW

**Comment by dbadoy** [comment]
Additional) Here is list used the  `docs.ipfs.io`.
They need change to `docs.ipfs.tech` ?
```
.github/ISSUE_TEMPLATE/config.yml:13:   url: https://docs.ipfs.io/reference/http/api/
.github/ISSUE_TEMPLATE/doc.yml:10:        Problems with documentation on https://docs.ipfs.io should be reported to https://github.com/ipfs/ipfs-docs
.github/ISSUE_TEMPLATE/doc.yml:16:        - label: I am reporting a documentation issue in this repo, not https://docs.ipfs.io.
README.md:17:- extensive [command line tooling](https://docs.ipfs.io/reference/cli/)
README.md:21:Note: [other implementations exist](https://docs.ipfs.io/basics/ipfs-implementations/).
README.md:27:For more info see: https://docs.ipfs.io/introduction/overview/
README.md:35:  - Or [chat with us](https://docs.ipfs.io/community/chat/).
README.md:100:The canonical download instructions for IPFS are over at: https://docs.ipfs.io/guides/guides/install/. It is **highly recommended** you follow those instructions if you are not interested in working on IPFS development.
README.md:118:More info on how to run kubo (go-ipfs) inside Docker can be found [here](https://docs.ipfs.io/how-to/run-ipfs-inside-docker/).
README.md:366:[![docs: Command-line quick start](https://img.shields.io/static/v1?label=docs&message=Command-line%20quick%20start&color=blue&style=flat-square&cacheSeconds=3600)](https://docs.ipfs.io/how-to/command-line-quick-start/)
README.md:367:[![docs: Command-line reference](https://img.shields.io/static/v1?label=docs&message=Command-line%20reference&color=blue&style=flat-square&cacheSeconds=3600)](https://docs.ipfs.io/reference/cli/)
README.md:390:If you believe you've found a bug, check the [issues list](https://github.com/ipfs/kubo/issues) and, if you don't see your problem there, either come talk to us on [Matrix chat](https://docs.ipfs.io/community/chat/), or file an issue of your own!
README.md:394:See [IPFS in GO](https://docs.ipfs.io/reference/go/api/) documentation.
README.md:449:Please reach out to us in one [chat](https://docs.ipfs.io/community/chat/) rooms.
core/corehttp/hostname_test.go:94:		{"docs-ipfs-io", "docs.ipfs.io"},
docs/README.md:3:If you are looking for User Documentation & Guides, please visit [docs.ipfs.io](https://docs.ipfs.io/).
docs/changelogs/v0.10.md:34:  * `dag get` now takes the `output-codec` option which accepts a [multicodec](https://docs.ipfs.io/concepts/glossary/#multicodec) name used to encode the output. By default this is `dag-json`, which is  a strict and deterministic subset of JSON created by the IPLD team. Users may notice differences from the previously plain Go JSON output, particularly where bytes are concerned which are now encoded using a form similar to CIDs: `{"/":{"bytes":"unpadded-base64-bytes"}}` rather than the previously Go-specific plain padded base64 string. See the [dag-json specification](https://ipld.io/specs/codecs/dag-json/spec/) for an explanation of these forms.
docs/changelogs/v0.10.md:50:go-ipfs now provides utility commands for working with [multibase](https://docs.ipfs.io/concepts/glossary/#multibase):
docs/changelogs/v0.11.md:34:   - HAMT-based sharding is applied to large directories (i.e. those that would serialize into [block](https://docs.ipfs.io/concepts/glossary/#block) larger than ~256KiB)s. This means importing data via commands like `ipfs add -r <directory>` may result in different [CID](https://docs.ipfs.io/concepts/glossary/#cid)s due to the different [DAG](https://docs.ipfs.io/concepts/glossary/#dag) representations.
docs/changelogs/v0.11.md:36:- go-ipfs can no longer act as a [Circuit Relay](https://docs.ipfs.io/concepts/glossary/#circuit-relay) v1
docs/changelogs/v0.11.md:49:Truly big directories can have so many items, that the root block with all of their names is too big to be exchanged with other peers. This was partially solved by [HAMT-sharding](https://docs.ipfs.io/concepts/glossary/#hamt-sharding), which was introduced a while ago as opt-in. The main downside of the implementation was that it was a global flag that sharded all imported directories (big and small).
docs/changelogs/v0.11.md:57:This is the cornerstone for maximizing p2p connections between IPFS peers. Every publicly dialable peer can now act as a limited relay v2, which can be used for [hole punching](https://docs.ipfs.io/concepts/glossary/#hole-punching) and other decentralized signaling protocols.
docs/changelogs/v0.11.md:96:If you use `/api/v0/pubsub/*` directly or maintain your own client library, you must adjust your HTTP client code. Byte fields and URL args are now encoded in `base64url` [Multibase](https://docs.ipfs.io/concepts/glossary/#multibase). Encode/decode bytes using the `ipfs multibase --help` commands, or use the multiformats libraries ([js-multiformats](https://github.com/multiformats/js-multiformats#readme), [go-multibase](https://github.com/multiformats/go-multibase)).
docs/changelogs/v0.12.md:71:We are switching the default low level [datastore](https://docs.ipfs.io/concepts/glossary/#datastore) to be keyed only by the [Multihash](https://docs.ipfs.io/concepts/glossary/#multihash) part of the [CID](https://docs.ipfs.io/concepts/glossary/#cid), and deduplicate some [blocks](https://docs.ipfs.io/concepts/glossary/#block) in the process. The blockstore will become [codec](https://docs.ipfs.io/concepts/glossary/#codec)-agnostic.
docs/changelogs/v0.13.md:9:and the [`v0/dag/import`](https://docs.ipfs.io/reference/http/api/#api-v0-dag-import) endpoint.
docs/changelogs/v0.13.md:218:*Why these two formats?* Requesting Block or CAR for `/ipfs/{cid}` allows a client to **use gateways in a trustless fashion**. These types of gateway responses can be verified locally and rejected if digest inside of requested CID does not match received bytes. This enables creation of "light IPFS clients" which use HTTP Gateways as inexpensive transport for [content-addressed](https://docs.ipfs.io/concepts/content-addressing/) data, unlocking use in Mobile and IoT contexts.
docs/changelogs/v0.13.md:283:This additional information allows HTTP caches and CDNs to make better decisions around cache invalidation: not just invalidate everything under specific IPNS website when the root changes, but do more fine-grained cache invalidation by detecting when only a specific subdirectory (branch of a [DAG](https://docs.ipfs.io/concepts/glossary/#dag)) changes.
docs/changelogs/v0.13.md:371:- Documentation: [ Run IPFS inside Docker](https://docs.ipfs.io/how-to/run-ipfs-inside-docker/) <!-- TODO: needs https://github.com/ipfs/ipfs-docs/pull/1115/files -->
docs/changelogs/v0.13.md:376:https://docs.ipfs.io/reference/http/api/ now includes separate sections for _experimental_ and _deprecated_ commands.
docs/changelogs/v0.14.md:73:Kubo now supports [`base256emoji`](https://github.com/multiformats/multibase/blob/master/rfcs/Base256Emoji.md) encoding in all [Multibase](https://docs.ipfs.io/concepts/glossary/#multibase) contexts. Use it for testing Unicode support, as visual aid while explaining Multiformats, or just for fun:
docs/changelogs/v0.9.md:162:IPFS community moved towards dedicated Origins (DNSLink and [subdomain gateways](https://docs.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway)) which are much easier to isolate and reason about.
docs/config.md:751:- `true` - enables [subdomain gateway](#https://docs.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway) at `http://*.{hostname}/`
docs/config.md:767:- `false` - enables [path gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#path-gateway) at `http://{hostname}/*`
docs/config.md:824:* Public [subdomain gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway) at `http://{cid}.ipfs.dweb.link` (each content root gets its own Origin)
docs/config.md:848:* Public [path gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#path-gateway) at `http://ipfs.io/ipfs/{cid}` (no Origin separation)
docs/config.md:864:* Hardened, site-specific [DNSLink gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#dnslink-gateway).
docs/config.md:1934:Options for configuring DNS resolution for [DNSLink](https://docs.ipfs.io/concepts/dnslink/) and `/dns*` [Multiaddrs](https://github.com/multiformats/multiaddr/).
docs/gateway.md:7:[More about Gateways](https://docs.ipfs.io/concepts/ipfs-gateway/) and [addressing IPFS on the web](https://docs.ipfs.io/how-to/address-ipfs-on-web/).
docs/gateway.md:14:a [path gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#path-gateway) at `http://127.0.0.1:8080/`
docs/gateway.md:15:and a [subdomain gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway) at `http://localhost:8080/`
docs/gateway.md:56:[DNSLink](https://docs.ipfs.io/concepts/glossary#dnslink). See [Example: IPFS
misc/systemd/ipfs-hardened.service:21:Documentation=https://docs.ipfs.io/
misc/systemd/ipfs.service:17:Documentation=https://docs.ipfs.io/

```

**Review by lidel — COMMENTED** [review]
@dbadoy  yes, all places with `docs.ipfs.io` should be replaced with `docs.ipfs.tech` – do you mind adding that to this PR? :pray:

**Comment by dbadoy** [comment]
ok, I'll commit that. 😃

**Comment by dbadoy** [comment]
@lidel  review please.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ contact_links:
    url: https://github.com/ipfs/kubo/blob/master/docs/experimental-features.md#readme
    about: Documentation on Private Networks, Filestore and other experimental features.
  - name: RPC API Reference
-   url: https://docs.ipfs.io/reference/http/api/
+   url: https://docs.ipfs.tech/reference/http/api/
```
Comment (lidel) [inline .github/ISSUE_TEMPLATE/config.yml:13]:
```suggestion
   url: https://docs.ipfs.tech/reference/kubo/rpc/
```

File: docs/changelogs/v0.13.md [hunk 2]
```diff
@@ -368,12 +368,12 @@ Docker images published at https://hub.docker.com/r/ipfs/go-ipfs/  now support c
...
 
 For more information, see:
-- Documentation: [ Run IPFS inside Docker](https://docs.ipfs.io/how-to/run-ipfs-inside-docker/) <!-- TODO: needs https://github.com/ipfs/ipfs-docs/pull/1115/files -->
+- Documentation: [ Run IPFS inside Docker](https://docs.ipfs.tech/how-to/run-ipfs-inside-docker/) <!-- TODO: needs https://github.com/ipfs/ipfs-docs/pull/1115/files -->
 - Examples in [ipfs-shipyard/go-ipfs-docker-examples](https://github.com/ipfs-shipyard/go-ipfs-docker-examples).
 
 #### RPC API docs for experimental and deprecated commands
 
-https://docs.ipfs.io/reference/http/api/ now includes separate sections for _experimental_ and _deprecated_ commands.
+https://docs.ipfs.tech/reference/http/api/ now includes separate sections for _experimental_ and _deprecated_ commands.
```
Comment (lidel) [inline docs/changelogs/v0.13.md:376]:
```suggestion
https://docs.ipfs.tech/reference/kubo/rpc/ now includes separate sections for _experimental_ and _deprecated_ commands.
```

File: docs/gateway.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ for details</sub>
 ## Static Websites
 
 You can use an IPFS gateway to serve static websites at a custom domain using
-[DNSLink](https://docs.ipfs.io/concepts/glossary#dnslink). See [Example: IPFS
+[DNSLink](https://docs.ipfs.tech/concepts/glossary/#dnslink). See [Example: IPFS
 Gateway](https://dnslink.io/#example-ipfs-gateway) for instructions.
```
Comment (lidel) [inline docs/gateway.md:57]:
```suggestion
Gateway](https://dnslink.dev/#example-ipfs-gateway) for instructions.
```

**Review by lidel — APPROVED** [review]
Thanks!

**Pull request merged**